login/login.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    </head>
+    <body>
+        <noscript>To play CardShifter, please either enable JavaScript or use a JavaScript-enabled browser.</noscript>
+        <form name = "login_information">
+            Server:
+            <input name = "server" type = "text" placeholder = "dwarftowers.com" />
```

Comment:
@Phrancis Yes, originally I had an option in the form for a port, but then I realized that `ws://` will always be on a single port, like how `http://` is always on port 80.